content/doc/book/blueocean/index.adoc

```diff
@@ -101,8 +101,8 @@ revisit how Jenkins currently expresses these concepts and consider delivery
 pipelines as a central theme to the Jenkins user experience.
 
 It is not just CD concepts but the tools that developers use
-every day – Github, Bitbucket, Slack, HipChat, Puppet or Docker. It is about
-more than Jenkins – it is the developer workflow which surrounds Jenkins that
+every day – Github, Bitbucket, Slack, HipChat, Puppet, or Docker. It is about
+more than Jenkins – it is the developer workflow that surrounds Jenkins that
```

Comment:
Combining two uses of "that" in the same sentence feels difficult to understand. Is there an alternate phrasing to avoid the duplication and still retain the meaning?
```suggestion
more than Jenkins – it is the developer workflow that surrounds Jenkins that
```

Reply:
Is this ok ?
it is the developer workflow surrounding Jenkins that spans multiple tools.

Reply:
> Combining two uses of "that" in the same sentence feels difficult to understand. Is there an alternate phrasing to avoid the duplication and still retain the meaning?

Yes, that sounds much better.  Thanks for the improvement!

Reply:
> Is this ok ? it is the developer workflow surrounding Jenkins that spans multiple tools.

Yes, that sounds much better. Thanks for the improvement!

Reply:
Done